test/integration/submission/test.data.ts

```diff
@@ -0,0 +1,27 @@
+export const ABCD_REGISTRATION_DOC = {
```

Comment:
you can put this in the same test no need for different file unless you have alot of these

Reply:
also define this as ABCD_REGISTRATION_DOC: ActiveRegistration to get type check here